Fix contract interfaces & asset states (depending on user asset combination v1 vs v2)

nabioz opened the pull request.

Commit:
Fix contract interfaces

Commit:
Fix typo

Commit:
Disable zaps and remove new icon

appleseed-iii reviewed (commented):
tested:
- [x] stake
- [x] unstake
- [x] wrap to gohm
- [x] unwrap from gohm
- [x] no assets goes to stake-v2

appleseed-iii committed:
state updates
1. add sOHMv2 to `newAssetsDetected` logic
2. add sOHMv2 balance to v1-stake (in case user has it & is waiting on bonds)
3. add `goToBonds` prompt on v1-stake if user has v1 bonds to claim before migration

appleseed-iii committed:
update contract addresses

appleseed-iii committed:
fix unwrap pending status

appleseed-iii committed:
remove old console log
